src/edu/csus/ecs/pc2/clics/API202003/ClarificationService.java

```diff
@@ -51,7 +50,7 @@ public ClarificationService(IInternalContest inContest, IInternalController inCo
     /**
      * This method returns a representation of the current contest groups in JSON format. The returned value is a JSON array with one language description per array element, matching the description
```

Comment:
The above comment says that this method returns information on GROUPS... and then says it returns an array containing LANGUAGES... and the method NAME is `getClarification()`...

Reply:
Again, old code base.   Do we want to go through and change those, even though the code really hasn't changed (just relocated?)  If you see any of these in the 202306 code base, then yes.  These look like the result of duplicating modules when the code was originally written.

Reply:
See above; I agree we should fix these in old code but only if we happen to be editing the files anyway...